sources/api/netdog/systemd-derive/README.md

```diff
@@ -0,0 +1,129 @@
+# systemd-derive
```

Comment:
I may benefit from this crate. Would it make sense to make it its own lib in `sources` out of `api`? We could name the crate `systemd-lib` and extend it as needed. I already know I want to have a function to receive the file descriptors passed by systemd, and I was planning to have this as a standalone lib that may be used by other “dogs”. We will use this both in EFS utils replacement, and when we decide to implement socket activation for API server.

Reply:
I don't mind if the macros move out of `netdog`, but I'd rather not conflate the these macros as part of a larger systemd-related "library" crate.